docs/deploy_narrative.md

```diff
@@ -1,64 +1,109 @@
 ## Narrative Development & Deployment HowTo
 
+This document describes how to release and deploy the Narrative Interface app onto the different KBase environments. It's intended for KBase developers and admins.
 
-### Overview
+- Last modified: Sep. 25, 2020
 
-1. Create a "development" image for CI.
-1. Deploy image to CI.
-1. Once tested in CI, create a single "production" image for appdev, next, & prod.
-1. Tag the "production" image for deployment using the `deploy_tag.sh` script.
-1. Deploy the "production" image in the relevant environments.
+**_Table of Contents_**
 
-### CI
+-   [Deployment flow overview](#deployment-flow-overview)
+-   [Deploying CI](#deploying-ci)
+    -   [Create CI Image](#create-ci-image)
+    -   [Release CI](#release-ci)
+-   [Stage a Final Release](#stage-a-final-release)
+-   [Deploying Next, Appdev, and Prod](#deploying-next,-appdev-&-prod)
+    -   [Create Release Image](#create-release-image)
+    -   [Tag & Deploy Image](#tag-&-deploy-image)
+-   [Image URLs](#image-urls)
+
+### Deployment Flow Overview
+
+1.  Create a "development" image for CI.
+2.  Deploy image to CI.
+3.  Once tested in CI, draft a release by bumping the version and creating a Github release tag.
+4.  Create a single production image for appdev, next, & prod.
+5.  Tag the production image for deployment using the `deploy_tag.sh` script.
+6.  Deploy the production image in the relevant environments.
+
+### Deploying CI
+
+The Continuous Integration environment - <https://ci.kbase.us> - is intended for testing and demoing of new features that may not always work right. It's also intended to be continuously upgraded and redeployed, so this process is designed to be as easy as possible.
 
 #### Create CI Image
 
-1. Create a new temporary branch from the `develop` branch.
-1. Add a _single_ bug fix or new feature to your temporary branch.
-1. Create a new [pull request](https://github.com/kbase/narrative/compare) to merge your branch into `develop`.
-1. Ensure the automated [build test](https://github.com/kbase/narrative/actions?query=workflow%3A%22Build+Test+Image%22) completes successfully.
-1. Complete review and testing.
-1. Merge pull request.
+1.  Create a new temporary branch from the `develop` branch.
```

Comment:
This title is a little confusing, given what's in the section below -- I was expecting instructions on building a docker image, but it's really the whole process of putting out a new release on CI. Perhaps just editing the title to something like `Creating a new image for CI`?
I don't think the two headers, `Create` and `Release`, are really necessary -- the `Release` section definitely seems surplus to requirements; the header could just be removed.

Reply:
Yeah, good idea. I originally had that separate because some instructions were a little ambiguous. With some other edits, it clarified a bit, so the subheadings aren't necessary.